modules/vaos/spec/request/appointments_request_spec.rb

```diff
@@ -4,12 +4,10 @@
 RSpec.describe 'vaos appointments', type: :request do
   include SchemaMatchers
 
-  let(:rsa_private) { OpenSSL::PKey::RSA.generate 4096 }
-
   before do
     Flipper.enable('va_online_scheduling')
     sign_in_as(current_user)
-    allow_any_instance_of(VAOS::JWT).to receive(:rsa_private).and_return(rsa_private)
+    allow_any_instance_of(VAOS::UserService).to receive(:session).and_return('stubbed_token')
```

Comment:
Nitpick: see this in a couple of the specs and there will likely be more in the future, possibly introduce rails helper or even shared examples where prudent. Can do as cleanup / refactor later.